lib/addons/cluster-autoscaler/index.ts

```diff
@@ -62,8 +64,12 @@ export class ClusterAutoScalerAddOn extends HelmAddOn {
 
         const cluster = clusterInfo.cluster;
         const nodeGroups = assertEC2NodeGroup(clusterInfo, "Cluster Autoscaler");
+        const values = this.options.values || {};
+        const namespace = this.options.namespace!;
+        const createANamespace = (this.options.namespace == "kube-system") ? false : true;
```

Comment:
A typical approach from the customers is to create a namespace like `addons` and put all those installed by the platform in there. This will not work in such cases, since the namespace will be reused across multiple add-ons. This is a bigger problem though, not only applicable in this case. 

The correct approach is:
1. add field `createNamespace` to the properties.
2. If set to true - create, otherwise assume it is already there.

minor: createANamespace => createNamespace